lib/geojsonLayerController.js

```diff
@@ -62,6 +65,7 @@ function buildConfigProcessingOptions(layer, config) {
     iconStylePresenter: config.stylePresenterFactory(nn(layer)('styles.icon').val),
```

Comment:
No idea what you are talking about.. Oh look up there what's that? An eagle?

On Thu, Jan 22, 2015 at 10:51 AM, Mark Stuart <notifications@github.com>
wrote:

>> @@ -62,6 +65,7 @@ function buildConfigProcessingOptions(layer, config) {
>>      iconStylePresenter: config.stylePresenterFactory(nn(layer)('styles.icon').val),
> You mean this module that is used just a few lines below? :stuck_out_tongue_winking_eye: 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/mediasuitenz/mappy/pull/77/files#r23337643